### ISSUE TITLE
[Firefox Bug] Firefox extensions have been disabled

### ISSUE THREAD

**Issue by trickypr** [joelshepherd/tabliss#133]
**This is not an issue with Tabliss. It is a problem with Firefox's add-on system**
# What is going on
Every add-on on the Firefox add-on store has been signed with a certificate. One of those certificates (a highly used one) has expired today. [Mozilla have a fix that they are currently testing](https://mobile.twitter.com/mozamo/status/1124484255159971840), although it might be another few hours before it is released. 

# Temporary Fix
Read the blog post linked in @TobiasFeld22 comment.
Do not attempt any of the temporary fixes in this [reddit article](https://www.reddit.com/r/firefox/comments/bkhtv8/heres_whats_going_on_with_your_addons_being/?utm_source=share&utm_medium=ios_app) about how to get everything working again.

**Comment by TobiasFeld22** [comment]
Mozilla has released an blog post with information on how to get things working again.
https://blog.mozilla.org/addons/2019/05/04/update-regarding-add-ons-in-firefox/

tldr
They are currently deploying a hotfix using the "studies" system, check if you have it enabled. 
It can take up to 6 hours to deploy the fix. 
The fix will also be applied in the next release that will be deployed in the next following days (follow the blog for details).

**Comment by trickypr** [comment]
Mozilla has released [an update with the fix](https://twitter.com/firefox/status/1125135184343396352?s=20). Everyone should have addons available again.
